src/Propel/Runtime/Collection/Collection.php

```diff
@@ -310,25 +310,25 @@ public function set($key, $value)
      * Alias for ArrayObject::offsetUnset()
      *
      * @param  mixed $key
-     * @return mixed The removed element
+     * @return void
      */
     public function remove($key)
     {
         if (!$this->offsetExists($key)) {
             throw new UnexpectedValueException(sprintf('Unknown key %s.', $key));
         }
 
-        return $this->offsetUnset($key);
```

Comment:
Are you changing API?

Reply:
Ah, it doesn't work.

Reply:
No, void code cannot return anything. So no chance in behavior.